packages/web/sentry.client.config.ts

```diff
@@ -4,14 +4,30 @@
 
 import * as Sentry from "@sentry/nextjs";
 
+import { getValidSwapTRPCRoutesForSentry } from "~/utils/sentry-init";
+
 Sentry.init({
   dsn: "https://c696452bb7ce4cc98150142ebea1c32f@o4505285755600896.ingest.us.sentry.io/4505285757698048",
 
   environment:
     process.env.NEXT_PUBLIC_VERCEL_ENV || process.env.NODE_ENV || "development",
 
   // Adjust this value in production, or use tracesSampler for greater control
-  tracesSampleRate: 0.05,
+  tracesSampler: (samplingContext) => {
+    const validPaths = ["/", "/pool/[id]", "/assets/[denom]"];
```

Comment:
Which one of these guarantees that `useSwap` hook is visible?

Reply:
The `useSwap` uses this file. All errors reported in the browser will be forwarded to Sentry.

Reply:
We previously discussed browser causing too much noise. Just asking for context - have we gotten a chance to address this?

Reply:
Thanks for reminding me. I'll reduce the error reporting sample rate to 25%. We'll revisit this once we make some improvements to reduce the noise coming from Cosmos-Kit